_data/datasets/contributed.yaml

```diff
@@ -1,741 +0,0 @@
-#########################################
```

Comment:
I think this will be replaced on the next data build, but whatever - I'll delete when I fix the builder

Reply:
Right, wasn't sure whether it will be so just deleted it on my side as well.